middlewares/articleValidation.js

```diff
@@ -22,6 +22,24 @@ export default {
     return next();
   },
 
+  categoryQueryValidator(req, res, next) {
+    const cate = [
```

Comment:
Import the `categoriesEnum` from the `helpers` directory and use it here instead of this `cate`.

Also, towards next time, a more descriptive name will suffice.